packages/ui-react/src/components/VideoDetails/__snapshots__/VideoDetails.test.tsx.snap

```diff
@@ -13,7 +13,7 @@ exports[`<VideoDetails> > renders and matches snapshot 1`] = `
         class="_main_d0c133 _mainPadding_d0c133"
       >
         <img
-          alt="Test video"
+          alt=""
```

Comment:
Weird that this test did not fail in PR: https://github.com/Videodock/ott-web-app/pull/68
Updated the snapshot eitherway.